lib/control.js

```diff
@@ -2,19 +2,12 @@
 
 const common = require('@metarhia/common');
 
-const { each } = require('./array');
-
 // Executes all asynchronous functions and pass first result to callback
 //   fns - <Function[]>, callback-last / err-first
 //   callback - <Function>, on done, err-first
 const firstOf = (fns, callback) => {
   const done = common.once(callback);
-  each(fns, (f, iterCb) =>
-    f((...args) => {
-      done(...args);
-      iterCb(...args);
-    })
-  );
+  fns.forEach(fn => fn((...args) => done(...args)));
```

Comment:
Why is this relevant?

Reply:
If new `async` `each` is used here, it will cause `setTimeOut` test error. Old one was `sync`.

Reply:
This seems more like an issue with tests.